design-system/ui/Header.tsx

```
@@ -10,30 +10,42 @@ interface HeaderProps {
   leftButtonLabel?: ReactNode; // 왼쪽 콘텐츠 (버튼)
   leftButtonClick?: () => void; // 왼쪽 버튼 클릭 핸들러
   leftButtonClassName?: string; // 왼쪽 버튼 추가 스타일링 클래스
-  rightContent?: Button; // 오른쪽 콘텐츠 (버튼)
+  rightContent?: Button | null; // 오른쪽 콘텐츠 (버튼)
   color?: 'white' | 'black'; // white 또는 black 중 선택
   className?: string; // 추가 스타일링 클래스
 }
 
+// Button 타입 가드 함수 추가
+function isButtonElement(element: any): element is React.ReactElement<ButtonHTMLAttributes<HTMLButtonElement>> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)